components/financial_assistance/app/models/financial_assistance/application.rb

```diff
@@ -640,6 +640,7 @@ def applicant_relative_exists_for_relations
 
     def validate_relationships(matrix)
       # validates the child has relationship as parent for 'spouse of the primary'.
```

Comment:
We should update this method description to acknowledge your check also for readability purposes